api/hrlh.rb

```diff
@@ -27,23 +36,41 @@ def database
         requires :email, type: String
       end
       post do
-        database['interviewer']
-        .set(
-            params[:email],
+        interview_db
+        .create(
```

Comment:
Place the . on the previous line, together with the method call receiver.<br>Use 2 (not 0) spaces for indenting an expression spanning multiple lines.